mitiq/executor/executor.py

```diff
@@ -119,24 +108,16 @@ def can_batch(self) -> bool:
 
         return return_type in (
             BatchedType[T]  # type: ignore[index]
-            for BatchedType in [
-                Iterable,
-                List,
-                Sequence,
-                Tuple,
-                list,
-                tuple,
-                collections.abc.Sequence,
-            ]
+            for BatchedType in [Iterable, list, Sequence, tuple, list, tuple]
```

Comment:
As above, lets keep this unchanged.

Reply:
Yes please! I know it may seem strange, but it's because we are inspecting type hints at runtime and need to support `typing` module type hints.